pom.xml

```diff
@@ -75,7 +75,7 @@
         <url>git@github.com:diennea/carapaceproxy.git</url>
         <connection>scm:git:${project.scm.url}</connection>
         <developerConnection>scm:git:${project.scm.url}</developerConnection>
-        <tag>release/1.11</tag>
+        <tag>HEAD</tag>
```

Comment:
`tag` value should be managed by the `maven-release-plugin` goals, and describe what actually is the commit status:

- `HEAD` on master
- `release/<x>.<y>` on release branches
- `v<x>.<y>.<z>` on tagged commit of release branch, that are the actually released points in history